### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ogidig5**
This PR automatically closes this issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,22 @@
...
+
+Implement a proof-of-concept (POC) for an AI-assisted tool capable of suggesting docstrings and improvements for existing code-base functions and modules. The tool should generate PRs that undergo review by students and tech leads to ensure accuracy and usefulness before being proposed to OSS maintainers for inclusion.
+
+### 2. AI-Assisted Test Coverage Improvement
+
+Develop a POC for an AI-assisted tool that reviews a codebase, analyzes existing test coverage, and generates new tests for uncovered code paths. The resulting PRs should create new tests, which are reviewed by students and tech leads before being suggested for inclusion in the codebase.
+
+### 3. AI-Assisted Issue Resolution
+
+Create a POC for an AI-assisted tool capable of fetching the codebase of an open-source project along with its existing issues and commentaries. The tool should solve each issue using a Language Model (LLM), potentially with a RAG architecture if necessary. The resulting PRs implementing solutions are reviewed by students and tech leads before being proposed to maintainers.
```
Comment (bmwoodruff) [inline README.md:17]:
```suggestion
Implement a POC for an AI-assisted tool capable of fetching the codebase of an open-source project along with its existing issues and commentaries, and then solve each issue using an LLM with a RAG architecture if necessary. The resulting PRs are reviewed by students and tech leads before being proposed to maintainers.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,22 @@
...
+
+## Examples of Useful PRs
+
+### 1. AI-Assisted Docstring Suggestions
+
+Implement a proof-of-concept (POC) for an AI-assisted tool capable of suggesting docstrings and improvements for existing code-base functions and modules. The tool should generate PRs that undergo review by students and tech leads to ensure accuracy and usefulness before being proposed to OSS maintainers for inclusion.
+
+### 2. AI-Assisted Test Coverage Improvement
+
+Develop a POC for an AI-assisted tool that reviews a codebase, analyzes existing test coverage, and generates new tests for uncovered code paths. The resulting PRs should create new tests, which are reviewed by students and tech leads before being suggested for inclusion in the codebase.
```
Comment (bmwoodruff) [inline README.md:13]:
```suggestion
Implement a POC for an AI-assisted tool that reviews a codebase, analyzes existing test coverage, and generates new tests for uncovered code paths. The resulting PRs should create new tests, which are reviewed by students and tech leads before being suggested for inclusion in the codebase.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,22 @@
...
+
+### 3. AI-Assisted Issue Resolution
+
+Create a POC for an AI-assisted tool capable of fetching the codebase of an open-source project along with its existing issues and commentaries. The tool should solve each issue using a Language Model (LLM), potentially with a RAG architecture if necessary. The resulting PRs implementing solutions are reviewed by students and tech leads before being proposed to maintainers.
+
+### 4. PR Guidelines Enforcement
```
Comment (bmwoodruff) [inline README.md:19]:
```suggestion
### 4. AI-Assisted PR Review
```

**Review by bmwoodruff — COMMENTED** [review]
Looks like you've got pull requests down. Adding the project description from our timeframe doc was a great idea. 

A couple things to practice.
1. Use the batching option to commit all three suggested changes into a single commit. 
2. Make a suggestion to change something else in this file.
3. Add an appropriate comment to address which issue this will close, so that GitHub appropriate connects this to the issue.

File: README.md [hunk 4]
```diff
@@ -1 +1,22 @@
...
\ No newline at end of file
+# Generative AI Project
+
+This project aims to develop AI-assisted tools that can enhance the code-base of open-source projects by generating useful pull requests (PRs). The project involves curation and oversight of students along with mentors to ensure the quality and effectiveness of the generated PRs.
+
+## Examples of Useful PRs
+
+### 1. AI-Assisted Docstring Suggestions
+
+Implement a proof-of-concept (POC) for an AI-assisted tool capable of suggesting docstrings and improvements for existing code-base functions and modules. The tool should generate PRs that undergo review by students and tech leads to ensure accuracy and usefulness before being proposed to OSS maintainers for inclusion.
```
Comment (ogidig5) [inline README.md:9]:
```suggestion
Implement a proof-of-concept (POC) for an AI-assisted tool capable of suggesting docstrings and improvements for existing code-base functions and modules. This tool will suggest docstrings and potential enhancements for existing functions and modules. The generated pull requests (PRs) will undergo a review process by students and tech leads to guarantee both accuracy and practical value before being presented for potential integration into open-source software (OSS) projects.
```

**Comment by ogidig5** [comment]
@bmwoodruff I have implemented the following suggested fixes:

1. Batching all 3 suggested changes into a single, well-described commit.
2. Adding my own suggested change.
3. Adding an appropriate comment that closes this issue.

This closes issue #6.

**Comment by bmwoodruff** [comment]
Looks like you got the process down!

**Pull request merged**